MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -444,22 +449,31 @@ public string GetAudioEncoder(EncodingJobInfo state)
         public string GetInputArgument(EncodingJobInfo state, EncodingOptions encodingOptions)
         {
             var arg = new StringBuilder();
+            var videoDecoder = GetHardwareAcceleratedVideoDecoder(state, encodingOptions);
+            var outputVideoCodec = GetVideoEncoder(state, encodingOptions);
 
-            if (state.IsVideoRequest
-                && string.Equals(encodingOptions.HardwareAccelerationType, "vaapi", StringComparison.OrdinalIgnoreCase))
+            if (state.IsVideoRequest && string.Equals(encodingOptions.HardwareAccelerationType, "vaapi", StringComparison.OrdinalIgnoreCase))
```

Comment:
I don't like this change. First of all, it's reversing the similar change from before. Second, it makes diff scroll horizontally while on github unified diff (not sure about split). Let's not do so.